companies/sbahn-berlin.json

```diff
@@ -0,0 +1,23 @@
+{
+    "slug": "sbahn-berlin",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "S-Bahn Berlin GmbH",
+    "address": "Elisabeth-Schwarzhaupt-Platz 1\n10115 Berlin\nDeutschland",
+    "phone": "+49 30 29743333",
+    "email": "Datenschutz.Regio@deutschebahn.com",
+    "sources": [
+        "https://sbahn.berlin/datenschutz/",
+        "https://sbahn.berlin/impressum/"
+    ],
+    "comments": [
```

Comment:
The comments are displayed fairly prominently in the generator, so we try to only include important ones. I think we can remove both of them in this case.